test_agasc1p7.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+import tables
```

Comment:
This test script should pass on Py2 and Py3, so also `import tables3_api`.  See https://github.com/sot/tables3_api/blob/master/README.md